vid/src/avid_m/proofs.rs

```diff
@@ -142,6 +249,7 @@ pub struct NsProof {
 
 impl NsAvidMScheme {
     /// Generate a proof of inclusion for a namespace payload.
+    /// WARN: for the current implementation, no proof can be generated if any namespace is malformed.
     pub fn namespace_proof(
```

Comment:
can you elaborate on this more? why is this acceptable? 
(we can discuss offline and not blocking this PR)